Doc/includes/sqlite3/shortcut_methods.py

```diff
@@ -18,3 +18,5 @@
     print(row)
 
 print("I just deleted", con.execute("delete from person").rowcount, "rows")
+
```

Comment:
Since this example is created to show the shortcut methods, perhaps we could add a short comment explaining that `close()` is not one of them.